tests/integration/framework/process/scheduler/scheduler.go

```diff
@@ -116,7 +129,11 @@ func New(t *testing.T, fopts ...Option) *Scheduler {
 	}
 
 	return &Scheduler{
-		exec:                exec.New(t, binary.EnvValue("scheduler"), args, opts.execOpts...),
+		exec: exec.New(t, binary.EnvValue("scheduler"), args,
+			append(opts.execOpts, exec.WithEnvVars(t,
```

Comment:
Does this set the namespace to nil if the env var is not set? Is that how our other tests run? I thought we default to `default` ns if unset

Reply:
I guess this is fine, I was looking at our `daprd.go` and assumed we should be consistent for ns: 
```
	ns := "default"
	if opts.namespace != nil {
		ns = *opts.namespace
		opts.execOpts = append(opts.execOpts, exec.WithEnvVars(t, "NAMESPACE", *opts.namespace))
	}
```
Then use the ns in the struct. Either way works really though